frequency: 10.0
base_frame_id: "base_footprint"
odom_frame_id: "odom"
transform_tolerance: 0.5
source_timeout: 5.0
base_shift_correction: True
polygons: [ "PolygonFront" ]
PolygonFront:
  # type可选： "polygon" "circle"
  type: "polygon"
  points: [ 0.3, 0.3, 0.3, -0.3, 0.0, -0.3, 0.0, 0.3 ]
  action_type: "none"
  min_points: 4
  visualize: True
  polygon_pub_topic: "polygon_front"
observation_sources: [ "scan" ]
scan:
  # type可选： "scan" "pointcloud" "range"
  type: "scan"
  topic: "scan"
pointcloud:
  type: "pointcloud"
  topic: "/intel_realsense_r200_depth/points"
  min_height: 0.1
  max_height: 0.5
